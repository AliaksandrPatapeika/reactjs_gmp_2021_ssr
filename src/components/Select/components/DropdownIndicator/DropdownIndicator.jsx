import React from 'react';
import {components} from 'react-select';

const DropdownIndicator = (props) => (
  <components.DropdownIndicator {...props}>
    <i className="fa fa-caret-down" aria-hidden="true" />
  </components.DropdownIndicator>
);

export default DropdownIndicator;
