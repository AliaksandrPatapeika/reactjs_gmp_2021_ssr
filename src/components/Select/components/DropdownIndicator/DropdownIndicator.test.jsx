import {noop} from 'lodash';
import React from 'react';
import renderer from 'react-test-renderer';

import DropdownIndicator from './DropdownIndicator';

describe('DropdownIndicator', () => {
  test('renders correctly', () => {
    const dropdownIndicator = renderer
      .create(<DropdownIndicator getStyles={noop} cx={noop} />)
      .toJSON();

    expect(dropdownIndicator).toMatchSnapshot();
  });
});
