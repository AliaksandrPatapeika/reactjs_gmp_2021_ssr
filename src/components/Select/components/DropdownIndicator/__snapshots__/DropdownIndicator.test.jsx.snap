// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownIndicator renders correctly 1`] = `
<div
  className="css-hbt8bd-DropdownIndicator"
>
  <i
    aria-hidden="true"
    className="fa fa-caret-down"
  />
</div>
`;
